### PR TITLE
windows compile HowTo

### PR DESCRIPTION
**Pull request by psychocrypt**
add documentation howto install all dependencies and compile with VS2017

To follow the instructions you need #160 as code base 

- [ ] must be merged after #160 
- [x] tests Windows 10
- [x] tests Windows 7

~~**WARNING** This pull request is currently under development . I wrote the documentation under linux and need to test all again in Windows.~~

@fireice-uk feel free to test it

### CODE REVIEW

**Comment by psychocrypt** [comment]
I verified the documentation on a Windows7 system too

File: WINCOMPILE.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+- download and unzip `xmr-stak-cpu`
+- open a command line `cmd`
+- `cd` to your unzipped source code directory
+- execute the following commands (NOTE: path to VS2017 can be different)
+  ```
+  "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\Tools\VsMSBuildCmd.bat"
+  set CMAKE_PREFIX_PATH=C:\xmr-stak-dep\hwloc-win64-build-1.11.7;C:\xmr-stak-dep\libmicrohttpd-0.9.55-w32-bin\x86_64\VS2017\Release-static
+  mkdir build
+  cd build
+  cmake -G "Visual Studio 15 2017 Win64" -T v141,host=x64 ..
```
Comment (fireice-uk) [inline WINCOMPILE.md]: I was unable to get this to work without extra `-DOpenSSL_ENABLE=OFF`. I don't think it is implemented in our cmake script so this should be added to the guide until finding SSL is supported too

Comment (psychocrypt) [inline WINCOMPILE.md]: With the linked openssl packages it builds without any issues on two different testsystems. Could you please post your OpenSSL folder structure.
Do you added the links to openSSL to CMake prefix path, If you are using another OpenSSL where the pathes are not globaly set than you need to add the path to camek.

Comment (fireice-uk) [inline WINCOMPILE.md]: Yup, custom compiled openssl was the problem. Got it working with a proper CMake path.

**Comment by MrT63** [comment]
I get an error message 

image
I installed only the tested packages on a win 7 sp1 system. directories all match.

Can you advise how to fix that>? Pls

**Comment by MrT63** [comment]
Sorry that is a useless pic ..see this
image

**Comment by MrT63** [comment]
I got no errors on installing Cmake with the 2 packages indicated.  Should i install other packages?

**Comment by MrT63** [comment]
Here is the log

[CMakeOutput.txt](https://github.com/fireice-uk/xmr-stak-cpu/files/1268961/CMakeOutput.txt)
[CMakeError.txt](https://github.com/fireice-uk/xmr-stak-cpu/files/1268962/CMakeError.txt)

**Comment by MrT63** [comment]
Why is it that you dont post a pre-compiled version for windows..would make life so much easier for the many miners who are not code geeks like me.  A version with 1% donation and one with 0.5% for example.  Please do that as i wasted several hourse trying to get this working and still cant mine using your platform and making you and me money

**Comment by psychocrypt** [comment]
@MrT63 you posted a amd miner issue in the cpu reposetory. Please open an issu in the amd reposetory.

A precompiled release can be found https://github.com/fireice-uk/xmr-stak-amd/releases

I will not follow all other posts to not mix up cpu and amd miner und this is an already merged feature, for problems please open an issue.
A precompiled version of amd is available https://github.com/fireice-uk/xmr-stak-amd/releases

**Comment by MrT63** [comment]
Thank you

**Pull request merged**